cves/kernel/CVE-2017-18379.yml

```diff
@@ -387,37 +409,37 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
-    applies:
+    applies: false
     note:
   least_privilege:
-    applies:
+    applies: false
     note:
   frameworks_are_optional:
-    applies:
+    applies: false
     note:
   native_wrappers:
-    applies:
+    applies: false
     note:
   distrust_input:
-    applies:
+    applies: false
```

Comment:
I may be wrong but this vulnerability may serve as distrust input lesson cause we get queue id from connection id which is input of the function.